src/types/models.ts

```diff
@@ -285,3 +285,11 @@ type Flatten<EventType extends SeamEvent["event_type"]> =
     : never
 
 export type Event = Flatten<SeamEvent["event_type"]>
+
+export interface ClientSessionI {
```

Comment:
🐧 we don't generally add `I`/`T` suffixes in this repo

Reply:
If we are going to introduce this, I think we should rename all interfaces to be consistent then.

Reply:
* we should match existing repo convention to avoid bikeshedding (same rules as open source), so shouldn't be there
* should be prefix rather than suffix to match other Seam repos that use this convention
* not sure there's a meaningful enough different between the interface and a type, the convention makes the most sense where classes are used. Since we almost never use classes (this SDK is a bit of an exception), it's usually unnecessary. Would be strongly opposed to adding a T prefix to types and to a lesser extent the I prefix, since it's typically already distinguished by PascalCase and there are no classes so that's enough
* If this comes up more often, linter convention is better